projects/packages/waf/actions.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Action Hooks for Jetpack WAF module.
+ *
+ * @package automattic/jetpack-waf
+ */
+
+namespace Automattic\Jetpack\Waf;
+
+// We don't want to be anything in here outside WP context.
+if ( ! function_exists( 'add_action' ) ) {
+	return;
+}
+
+/**
+ * Determines if the passed $option is one of the allowed WAF operation modes.
+ *
+ * @param  string $option The mode option.
+ * @return bool
+ */
+function jpwaf_is_allowed_mode( $option ) {
```

Comment:
Since we're under the Waf namespace, do we need the prefix in that function?

Reply:
@jeherve Follow-up PR created in #22896.